src/table_factory.h

```diff
@@ -45,15 +49,23 @@ class TitanTableFactory : public TableFactory {
 
   void* GetOptions() override { return base_factory_->GetOptions(); }
 
+  void SetBlobRunMode(TitanBlobRunMode mode) {
```

Comment:
Either add comment saying holding `TitanDBImpl::mutex_` lock is required, or add a mutex to guard `mutable_cf_options_`.